saleor/order/utils.py

```diff
@@ -39,6 +39,10 @@ def cancel_order(order):
         group.save()
 
 
+def create_history_entry(order, content, user=None):
+    order.history.create(content=content, user=user)
```

Comment:
Is there any reason that we do not call ```order.history.create(...)``` directly?

Reply:
That's a good point and something we were already talking about. I also don't see reasons to keep this function.

Reply:
Sure, removed.